data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaSourceConfig.java

```diff
@@ -19,6 +19,21 @@
  */
 
 public class KafkaSourceConfig {
+    public class SslConfig {
+        // TODO Add support for SSL Encryption by having
+        // path to certificates, etc
+    }
+
+    public class EncryptionConfig {
+        // Absence of ssl means no-encryption
```

Comment:
So this means PLAINTEXT? should we make it explicitly configurable. Customers may assume SSL by default if not specified.

Reply:
But SSL requires additional config like pointers to certificates, etc. Where as no-encryption does not require any config, right?

Reply:
Defaulted to SSL.